src/scheduler/TestScheduler.ts

```diff
@@ -22,15 +19,25 @@ class TestScheduler extends VirtualTimeScheduler {
   private flushing: boolean = false;
 
   private readonly _maxFrame: number;
-  public get maxFrame(): number {
-    return this._maxFrame;
-  }
 
   constructor(private readonly autoFlush: boolean, private readonly frameTimeFactor: number, maxFrameValue: number) {
     super(VirtualAction, Number.POSITIVE_INFINITY);
     this._maxFrame = maxFrameValue * frameTimeFactor;
   }
 
+  /*
+  public get maxFrame(): number {
+    return this._maxFrame;
+  }
+  */
+
+  // Use a non-accessor method to avoid the 'error TS1086: An accessor cannot be declared in an ambient context.'
+  // when used with versions of Typescript <= 3.5.
+  // See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#37-breaking-changes
+  public getMaxFrame(): number {
```

Comment:
This isn't ideal but to prevent breaking changes, I think this should be exposed as property instead:

```
public readonly maxFrame;

ctor(...maxFrameValue: number) { 
  this._maxFrame = maxFrameValue;
  this.maxFrame = this._maxFrame;
}
```